bootstrap/config.go

```diff
@@ -18,3 +20,24 @@ func LoadConfig() {
 		log.Fatal("Failed to load config: ", err)
 	}
 }
+
+func LoadENV() {
+	env := gin.Mode()
+
+	// Skip release mode because there is NO ENV file
+	if env == gin.ReleaseMode {
+		return
+	}
+
+	if env == gin.TestMode {
+		err := godotenv.Load(".env." + env)
+		if err != nil {
+			log.Fatal("Failed to load .env.", env, err)
+		}
+	}
+
+	err := godotenv.Load()
```

Comment:
Is it normal that we run that also for TestMode? 🤔 It seems incorrect as we already called `godotenv.Load(".env." + env)` a few lines earlier. But I might be wrong :)

Reply:
Oh after re read I can see the early return.